src/components/GetUniversalSearchResults.component.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Autosuggest from 'react-autosuggest'
+import { Icon } from './'
+
+export default class GetUniversalSearchResults extends Component {
+  onChange = (event, { newValue, method }) => {
+    newValue !== 'undefined'
+      ? this.props.updateUniversalSearchInput(newValue)
+      : this.props.updateUniversalSearchInput('')
+  }
+
+  onSuggestionSelected = (event, { suggestion }) => {
+    let mapYear = suggestion.mapYear
+    let res = this.props.maps.filter(m => m.year === mapYear)
+    this.props.setSelectedMap(res[0])
+    this.props.setSelectedPOI(suggestion)
+  }
+
+  onSuggestionsFetchRequested = () => {
+    this.props.universalSearchPOIs()
+  }
+
+  getPoiValue = poi => poi.name
```

Comment:
this is simple enough that you can just inline it instead of defining a new class function